### PR TITLE
Topology 3

### PR DESCRIPTION
**Pull request by jacobp100**
GitHub wasn't updating old PR - https://github.com/ag-grid/ag-charts/pull/1127

### CODE REVIEW

**Review by alantreadway — COMMENTED** [review]
https://sonarcloud.io/summary/new_code?id=ag-charts-community-latest&pullRequest=1137

**Comment by jacobp100** [comment]
> https://sonarcloud.io/summary/new_code?id=ag-charts-community-latest&pullRequest=1137

@alantreadway Fixed

File: packages/ag-charts-community/src/chart/series/topology/mapSeries.ts [hunk 1]
```diff
@@ -300,62 +406,77 @@ export class MapSeries extends DataModelSeries<
...
-
-    override async update(): Promise<void> {
-        const { itemSelection, labelSelection, markerSelection, highlightSelection } = this;
+    override async update({ seriesRect }: { seriesRect?: BBox }): Promise<void> {
+        const {
+            datumSelection,
+            labelSelection,
+            markerSelection,
+            highlightDatumSelection,
+            // highlightLabelSelection,
```
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/series/topology/mapSeries.ts:415]: Nit: Remove?

**Review by alantreadway — APPROVED** [review]
LGTM other than one nit

**Pull request merged**